### ISSUE TITLE
The otherway around

### ISSUE THREAD

**Issue by triDcontrols** [hobbyquaker/node-red-contrib-mqtt-json#1]
Hi, 

Is there something like this but the reverse, to take a JSON payload and publish to mqtt broker. 

Each object and array to be separated into separate topic. Attached is a pic of a JSON data that is large, and has many objects and arrays, and I would like to publish all this to a mqtt broker but be separated into it's own topics. 

![capture](https://user-images.githubusercontent.com/22411662/41391059-67c96720-6f4d-11e8-8efd-3231ccc61de0.PNG)

**Comment by hobbyquaker** [comment]
hmm... would you like to have always the first level of property keys to be the topics? so e.g.
```javascript
{
  "a": 1, 
  "b": 2, 
  "c": {
    "d": {
      "e": 3
    }
  }
}
```
would result in (topic - message)
* `a` - `1`
* `b` - `2`
* `c` - `{"d":{"e":3}}`
?

**Comment by triDcontrols** [comment]
Yes, that would be great with the ability to set a base mqtt topic.

**Comment by hobbyquaker** [comment]
that's quite easy to accomplish, but I think it's "out of scope" for mqtt-json. Will create a new project and name it "mqtt-multi-publish" or smth. Give me a few days.

**Comment by triDcontrols** [comment]
Awesome. Or just reverse the name. 
node-red-contrib-json-mqtt

**Comment by triDcontrols** [comment]
Hi, Any update? 

Thanks,
